client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -21,118 +20,164 @@ class PostRelativeTime extends React.PureComponent {
-			const scheduledTime = scheduledDate.calendar( null, {
-				sameElse: this.props.translate( 'll [at] LT', {
-					comment:
-						'll refers to date (eg. 21 Apr) & LT refers to time (eg. 18:00) - "at" is translated',
-				} ),
-			} );
-
-			const displayScheduleTime =
-				scheduledDate.diff( now, 'years' ) > 0
-					? scheduledTime
-					: scheduledTime.replace( scheduledDate.format( 'Y' ), '' );
-
-			statusText = this.props.translate( 'scheduled for %(displayScheduleTime)s', {
-				comment: '%(displayScheduleTime)s is when a scheduled post is set to be published',
+		} else if ( status === 'trash' ) {
+			statusClassName += ' is-trash';
+			statusIcon = 'trash';
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'trashed %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a post or page was trashed',
 				args: {
 					displayScheduleTime,
 				},
 			} );
+		} else if ( status === 'future' ) {
 			statusClassName += ' is-scheduled';
 			statusIcon = 'calendar';
-		} else if ( status === 'trash' ) {
-			statusText = this.props.translate( 'trashed' );
-			statusClassName += ' is-trash';
-			statusIcon = 'trash';
-		} else if ( this.props.includeBasicStatus ) {
-			if ( status === 'draft' ) {
-				statusText = this.props.translate( 'draft' );
-			} else if ( status === 'publish' ) {
-				statusText = this.props.translate( 'published' );
-			} else if ( status === 'new' ) {
-				statusText = this.props.translate( 'Publish immediately' );
-			}
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'scheduled %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a scheduled post or page is set to be published',
+				args: {
+					displayScheduleTime,
+				},
+			} );
+		} else if ( status === 'draft' || status === 'pending' ) {
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'draft last modified %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a draft post or page was last modified',
+				args: {
+					displayScheduleTime,
+				},
+			} );
+		} else if ( status === 'publish' ) {
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'published %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a post or page was last modified',
+				args: {
+					displayScheduleTime,
+				},
+			} );
+		} else if ( status === 'private' ) {
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'private last modified %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a private post or page was last modified',
+				args: {
+					displayScheduleTime,
+				},
+			} );
+		} else if ( status === 'new' ) {
```

Comment:
How can we test this status? Is it used for posts/pages that have been just published?

Reply:
I am not sure. I haven't been able to test it either. It is on master, though:

https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/post-relative-time-status/index.jsx#L133